stdlib/xml/etree/ElementTree.pyi

```diff
@@ -124,6 +124,8 @@ class Element:
     def __setitem__(self, __key: SupportsIndex, __value: Element) -> None: ...
     @overload
     def __setitem__(self, __key: slice, __value: Iterable[Element]) -> None: ...
+    @deprecated("Testing an element's truth value is deprecated.")
```

Comment:
This always behaved confusingly (and raised a FutureWarning under some circumstances), so seems fine to always warn.